keyboards/hhkb/keymaps/tominabox1/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.h
+matrix.c
+rules.mk
```

Comment:
This file should be removed.

Reply:
So I was using TMK years ago with a Teensy so I just copied some stuff over so that it worked. I'm not sure what is being used in Teensy. I'd like to make sure this works before pushing to the main, so thought I'd just hide them for now.